internal/config/cgmgr/cgroupfs.go

```diff
@@ -155,6 +155,17 @@ func setWorkloadSettings(cgPath string, resources *rspec.LinuxResources) (err er
 	return mgr.Set(cg.Resources)
 }
 
+// createSandboxCgroup takes the sandbox parent, and sandbox ID.
+// It creates a new cgroup for that sandbox, which is useful when spoofing an infra container.
+func createSandboxCgroup(sbParent, containerID string, mgr CgroupManager) error {
+	cgroupAbsolutePath, err := mgr.ContainerCgroupAbsolutePath(sbParent, containerID)
+	if err != nil {
+		return err
+	}
+	_, err = cgroups.New(cgroupAbsolutePath, &cgcfgs.Resources{})
```

Comment:
I'm just wondering why the if/else statements starting at line 150 in the old code were removed?

Reply:
Never mind, I found the explanation in the Issue.
LGTM